.changeset/rude-spoons-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/routing-utils': minor
```

Comment:
Is the value staying the same at the API? This might be a `major` version bump either way now that I think about it.

Reply:
Value stays the same - yes. Should we consider this breaking given it's going from less to more permissive? I would agree if it was the opposite. But I may not be thinking of a case when this could be considered breaking.